app/helpers/is-equal.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
+
+export function isEqual([a, b] /*, hash*/ ) {
+  return a == b;
```

Comment:
this won't return true equality. i think i've said this plenty of times now. please review == vs ===